src/guiguts/highlight.py

```diff
@@ -27,6 +28,14 @@ class Highlight:
         "Default": {"bg": "#a08dfc", "fg": "black"},
     }
 
+    TAG_SPOTLIGHT = str(auto())
+
+    COLORS_SPOTLIGHT = {
+        "Light": {"bg": "orange", "fg": "black"},
+        "Dark": {"bg": "orange", "fg": "white"},
+        "Default": {"bg": "orange", "fg": "black"},
```

Comment:
Aaah, this is why I see black text on orange in the main text window. :D 

Black looks fine on orange, which is a good reason to choose a color that black and white are equally readable on, because otherwise, fancy footwork would be needed with the default theme.

Reply:
See additional color notes in #353 relating to "fancy footwork" for default theme.